test/output/slim/document.html

```diff
@@ -0,0 +1,2194 @@
+<!-- .title -->
```

Comment:
I see now why the options were coming from the output document! Because I have no control on the input ones. Makes sense. Thanks for the pointer!